### PR TITLE
Use `filNam` parameter for TEASER loads and add within parsing to PackageParser

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
It appears to be customary to place filename variables as parameters at the top of the .mo files. For the TEASER workflow, we have been updating the path to the load files in the CombiTimeTable object directly. In addition, there were no methods for parsing out the within block of a package.mo file.

#### What does this PR accomplish?
* Moves the filename of the TEASER zones into a parameters at the top of the files
* Adds get and update methods to the package.mo parser
* Removes the <package_name> object when saving to the `DistrictEnergySystem.mo` file. In Dymola (and presumably in OMC and others), this is not needed and made downstream model manipulations a bit more difficult. 

#### How should this be manually tested?
* CI should suffice to catch the majority of the issues.
* Run test_teaser and then manually run the resulting the `teaster_single` model in Dymola.

#### What are the relevant tickets?
n/a

### CODE REVIEW

**Comment by nllong** [comment]
@vtnate  - can you look at this when you get back?

**Review by vtnate — APPROVED** [review]
Love it. Confirmed the model runs with OM.

Poetry is still occasionally choking when installing dependencies in CI. I just re-run the failed jobs until Poetry decides to work.

File: geojson_modelica_translator/modelica/package_parser.py [hunk 1]
```diff
@@ -37,6 +48,29 @@ class method.
...
+            self.within = None
+        elif len(matches) == 1:
+            self.within = matches[0].split(".")
+        else:
+            raise ValueError("There are more than on within statements in the package.mo file")
+
+        return self.within
+
+    def exists(self):
+        return self.path.exists()
```
[the remark this answers is not in the document]
Comment (nllong) [reply geojson_modelica_translator/modelica/package_parser.py:72]: hmm. i may have put this in there and then realized I didn't want it... I'll check.

[the remark this answers is not in the document]
Comment (nllong) [reply geojson_modelica_translator/modelica/package_parser.py:72]: i don't see it either, and I don't use it in the subsequent PRs #575 #577.  

I'll remove.

**Pull request merged**